signer/Cargo.toml

```diff
@@ -35,6 +40,9 @@ sr25519 = ["schnorrkel"]
 ecdsa = ["secp256k1"]
 unstable-eth = ["keccak-hash", "ecdsa", "secp256k1", "bip32"]
 
+# Enable support for loading key pairs from polkadot-js json.
+json = ["std", "subxt", "sr25519", "base64", "scrypt", "crypto_secretbox", "serde", "serde_json"]
```

Comment:
Do we really need `std` enabled for this `json` feature?

Reply:
This is necessary for the following command to succeed:
```
cargo test --no-default-features -F polkadot-js-compat
```

Reply:
Do you know why this is necessary for that command to succeed? I had another look over and it's not a huge deal but would be nice not to need the std feature :)